web/.env.local

```diff
@@ -0,0 +1,7 @@
+REEARTH_WEB_DEVELOPER_MODE=
+REEARTH_WEB_CESIUM_ION_TOKEN_URL=
+REEARTH_WEB_API=
+REEARTH_WEB_PLUGINS=
+REEARTH_WEB_AUTH0_DOMAIN=
+REEARTH_WEB_AUTH0_AUDIENCE=
+REEARTH_WEB_AUTH0_CLIENT_ID=
```

Comment:
Maybe we can provide the local urls as well as the template?

Reply:
Yes we can. Please share them or push directly to the PR.

Reply:
I add the url for first two, the rest can be added after we fix the local auth system i think